source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -18,13 +16,19 @@ Overview
 --------
 
 You can use read operations to retrieve data from your MongoDB database.
-There are multiple types of read operations that access the data in
-different ways. If you want to request results based on a set of criteria
-from the existing set of data, you can use a find operation such as the
-``find()`` or ``findOne()`` methods.
+To request results based on a set of criteria from the existing set of
+data, you can use a find operation such as the ``find()`` or
```

Comment:
S: not necessary, but I feel like saying "existing collection" is easier to understand than "existing set of data"(and avoids repeating "set")
```suggestion
To request results based on a set of criteria from an existing collection, you can use a find operation such as the ``find()`` or
```